gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageClientReadChannel.java

```diff
@@ -210,14 +212,41 @@ private class ContentReadChannel {
     // in-place seeks.
     private byte[] skipBuffer = null;
     private ReadableByteChannel byteChannel = null;
+    // Keeps track of distance between last 2 consecutive request.
+    private LimitedFifoQueue<Long> requestDistance = new LimitedFifoQueue<Long>(2);
```

Comment:
Queue "IS NOT" a Distance. Queue here is an implementation detail and we are leaking the abstraction here.

Reply:
point taken, will update the variable names here.